### ISSUE TITLE
Cannot use functions in view templates when calling sails.renderView()

### ISSUE THREAD

**Issue by jfrumar** [balderdashy/sails#2295]
When using res.view() with locals, you can pass in objects that reference functions (e.g. using [moment](http://momentjs.com/) to allow formatting of dates in the template).

If you call sails.renderView(), the line at:

https://github.com/balderdashy/sails/blob/f8870e796c22f5f9e39fb9d7c0047d73c82b157e/lib/hooks/views/render.js#L37

clones the options, but that removes any references to functions in the local vars that you may want to use in the EJS templates (so you can't use moment.format() to format your dates for example).

**Comment by CWyrtzen** [comment]
Is this resolved with a new release?

**Comment by sailsbot** [comment]
Thanks for posting, @jfrumar.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
